client/my-sites/plan-features/header.jsx

```diff
@@ -357,7 +386,7 @@ PlanFeaturesHeader.propTypes = {
 	billingTimeFrame: PropTypes.string.isRequired,
 	current: PropTypes.bool,
 	onClick: PropTypes.func,
-	planType: React.PropTypes.oneOf( [ PLAN_FREE, PLAN_PREMIUM, PLAN_BUSINESS ] ).isRequired,
+	planType: React.PropTypes.string.isRequired,
```

Comment:
Hmm, I think I was just getting tired last night. I can update to being the full list of plans.